app/views/unit_for_building/_edit_form.html.haml

```diff
@@ -3,10 +3,15 @@
   .form-inputs.margin-top-02
     = f.input :available, as: :boolean
     = f.input :name, label: "#{Unit.model_name.human} name"
-        
+    = f.input :occupancy, collection: 1..10, include_blank: false
```

Comment:
This seems small, but the collection should come from the unit `Unit.available_occupancies` which should also be used in the validation.  That way it all lines up if we decide to limit to 5 or increase to 15.  Same goes for the # of bedrooms below.